src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts

```diff
@@ -119,6 +119,10 @@ export class UbsUserProfilePageComponent implements OnInit, AfterViewInit, OnDes
     );
   }
 
+  checkedDefaultAddress(i): boolean {
```

Comment:
There is any reason to use this function?
And why don't implement setting default address with back-end in one time? (We discussed it on the daily)

Reply:
I removed this method